.github/ISSUE_TEMPLATE/monthly_reports.md

```diff
@@ -0,0 +1,87 @@
+# Checklist for monthly report generation
+
+## Process should being on the first of the month and reports emailed out no later than 2 weeks later
+
+### Creating report data
+
+``` python
+pip install -r requirements
+make generate_parameters
+make all -j 30
+```
+
+- [ ] verifying that parameters are generated in subfolders
+- [ ] verify in outputs that reports were generated
+- [ ] verify that data subfolder is present
+
+### Generate static site
+
+```python  
+npm run build
+python -m http.server
+```
+
+- [ ] report month has been added to UI
+- [ ] find example of agency with no validation errors (avalon), with validation notices (santa clara), and with daily metric errors (sacrt)
+
+### Review
```

Comment:
Either within or before review, there should be a checkbox for publishing to the development website.